src/components/templates/layout.tsx

```diff
@@ -8,8 +8,8 @@ const Layout: React.FC = ({ children }) => {
       <Sidebar />
       <div className="flex flex-col flex-1">
         <Topbar />
-        <div className="large:px-xlarge py-xlarge bg-grey-5 min-h-content overflow-y-scroll">
-          <main className="xsmall:mx-base small:mx-xlarge medium:mx-4xlarge large:mx-auto large:max-w-7xl large:w-full h-full">
+        <div className="large:px-xlarge py-xlarge bg-grey-5 min-h-content overflow-y-auto">
+          <main className="xsmall:mx-base small:mx-xlarge medium:mx-4xlarge large:mx-auto large:max-w-7xl large:w-full">
```

Comment:
I think this needs to be h-full acc. to the discussion in #234

Reply:
I think it is better to add this to the fix/two-split-pane PR instead, and not as part of this PR 😄